src/commands/to-arb/hierarchical-json-to-arb.converter.ts

```diff
@@ -0,0 +1,41 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as _ from "lodash";
+
+export class HierarchicalJsonToArbConverter {
```

Comment:
shouldn't this be called `HierarchicalJsonConverter`? Otherwise, why don't we move the `convertToArb` into the `ArbConverter` and rename to `fromHierarchicalJson`?

Reply:
I want to keep the conversion from ARB separated from the conversion to ARB, I think it'd be better to separate these topics, but I can do the renaming